bootleg/layers/alias_to_ent_encoder.py

```diff
@@ -82,7 +82,7 @@ def prep(
             log_rank_0_debug(logger, f"Loading alias table from {prep_file}")
             start = time.time()
             alias2entity_table = np.memmap(
-                prep_file, dtype="int64", mode="r", shape=data_shape
+                prep_file, dtype="int64", mode="r+", shape=data_shape
```

Comment:
Just trying to understand. Why was the r+ necessary. Thanks for fixing it!

Reply:
Opening with `r+` permits extending the file size to accommodate larger memory mapped files being loaded. It was needed when creating new alias files.